client/src/components/LoginSession.js

```diff
@@ -0,0 +1,110 @@
+/* global window */
+import React, { Component } from 'react';
+import confirm from '@silverstripe/reactstrap-confirm';
+import Config from 'lib/Config'; // eslint-disable-line
+import api from 'lib/api';
+import moment from 'moment';
+import i18n from 'i18n';
+import PropTypes from 'prop-types';
+
+class LoginSession extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            complete: false,
+            failed: false,
+            submitting: false,
+        };
+
+        this.logOut = this.logOut.bind(this);
+    }
+
+    async logOut() {
+        // const { ss: { i18n } } = window;
+
+        // Confirm with the user
+        const confirmMessage = i18n._t(
+            'SessionManager.DELETE_CONFIRMATION',
+            'Are you sure you want to delete this login session?'
+        );
+        const confirmTitle = i18n._t(
+            'SessionManager.CONFIRMATION_TITLE',
+            'Are you sure?'
+        );
+        const buttonLabel = i18n._t(
+            'SessionManager.DELETE_CONFIRMATION_BUTTON',
+            'Remove login session'
+        );
+
+        if (!await confirm(confirmMessage, { title: confirmTitle, confirmLabel: buttonLabel })) {
+            return;
+        }
+
+        this.setState({ submitting: true });
+
+        const token = Config.get('SecurityID');
+        const endpoint = `${this.props.LogOutEndpoint}/${this.props.ID}?SecurityID=${token}`;
+        api(endpoint, 'DELETE')
+            .then(response => response.json())
+            .then(output => {
+                const failed = !!output.error;
+
+                this.setState({ complete: true, failed, submitting: false });
+            })
+            .catch(() => {
+                this.setState({ complete: true, failed: true, submitting: false });
+            });
+    }
+
+    render() {
+        const lastAccessedElapsed = moment.utc(this.props.LastAccessed).fromNow();
+
+        if (this.state.complete || this.state.submitting) {
+            return null;
+        }
+
+        return (
+          <div>
+            <div>{this.props.UserAgent}</div>
+            {this.props.IsCurrent && <strong className={'text-success'}>{i18n._t(
+                    'SessionManager.CURRENT',
+                    'Current'
+                )}</strong>}
+            <div className="text-muted">
+              {this.props.IPAddress}
+              {!this.props.IsCurrent && `, ${i18n._t(
+                        'SessionManager.LAST_ACTIVE',
+                        'last active'
+                    )} ${lastAccessedElapsed}`}
+            </div>
+            {!this.props.IsCurrent && <a
+                  href="javascript:void(0);" // eslint-disable-line
```

Comment:
javascript:void(0) is pretty old fashioned, can we just remove this?

Reply:
Clicking this link should not redirect to the URL, but trigger the `onclick` event only.
This is the approach I ended up with after some research, if you have a better suggestion I'm all ears.